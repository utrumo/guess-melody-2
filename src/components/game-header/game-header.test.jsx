import React from 'react';
import renderer from 'react-test-renderer';
import GameHeader from './game-header.jsx';

it(`Game header correctly renders after relaunch`, () => {
  const gameHeader = renderer
    .create(<GameHeader />)
    .toJSON();

  expect(gameHeader).toMatchSnapshot();
});
