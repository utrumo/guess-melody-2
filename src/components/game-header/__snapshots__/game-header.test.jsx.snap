// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Game header correctly renders after relaunch 1`] = `
<header
  className="game__header"
>
  <a
    className="game__back"
    href="#"
  >
    <span
      className="visually-hidden"
    >
      Сыграть ещё раз
    </span>
    <img
      alt="Угадай мелодию"
      className="game__logo"
      src="img/melody-logo-ginger.png"
    />
  </a>
  <svg
    className="timer"
    viewBox="0 0 780 780"
    xmlns="http://www.w3.org/2000/svg"
  >
    <circle
      className="timer__line"
      cx="390"
      cy="390"
      r="370"
      style={
        Object {
          "filter": "url(#blur)",
          "transform": "rotate(-90deg) scaleY(-1)",
          "transformOrigin": "center",
        }
      }
    />
  </svg>
  <div
    className="timer__value"
    xmlns="http://www.w3.org/1999/xhtml"
  >
    <span
      className="timer__mins"
    >
      05
    </span>
    <span
      className="timer__dots"
    >
      :
    </span>
    <span
      className="timer__secs"
    >
      00
    </span>
  </div>
  <div
    className="game__mistakes"
  >
    <div
      className="wrong"
    />
    <div
      className="wrong"
    />
    <div
      className="wrong"
    />
  </div>
</header>
`;
